### DISCUSSION TITLE
Why rtp stream?

### DISCUSSION THREAD

**Discussion by Ncerzzk** [svpcom/wfb-ng#312] [Questions] [answered]
Hi, I just want to know:
what's the advantages of rtp stream with H264 payload  compared to   raw h264 udp stream?
less latency? or any other?

**Comment by svpcom** [comment, marked as answer]
From point of view wfb-ng is doesn't matter what is inside udp packets - it only forwards udp packets. But rtp is a standard and different programs can create and/or play it without interoperability issues

**Reply by Ncerzzk** [reply]
Thank you. I know that wfb-ng doesn't matter what is the payload. I'm just curious about it.
Actually I'm working on porting a new soc to OpenIPC now and feel curious about that if just directly send h264/h265 udp packets rather than rtp stream, what will lose

**Reply by Ncerzzk** [reply]
For example, as the QGC support udp h264 stream to play. Dose it mean that directly sending h264 udp packets to it and I don't need to do extra work in ground side?
Sorry that this may be out the box of wfb-ng.

**Reply by svpcom** [reply]
As I know QGC supports RTP h264/h265 streams. It use standard gstreamer pipelines inside

**Reply by Ncerzzk** [reply]
for now, qgc seems support h264/h265 udp streams too:https://docs.qgroundcontrol.com/master/en/SettingsView/General.html#video

**Reply by Ncerzzk** [reply]
sorry, It's my fault. The "UDP h.264 Video Stream" supported by QGC means "(RTP) UDP h.264 Streaming", not raw udp h.264 stream.
